ql/src/java/org/apache/hadoop/hive/ql/exec/FunctionUtils.java

```diff
@@ -126,6 +126,10 @@ public static String qualifyFunctionName(String functionName, String dbName) {
     } else if (names.length > 2) {
       throw new HiveException("Function name does not have correct format: " + functionName);
     }
+    // This is to fix the ranger related issue : HIVE-24182
```

Comment:
Can you edit this comment to be on the lines of "Remove the WINDOW_FUNC_PREFIX as that is not part of the database name"

Reply:
Done